packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -316,6 +317,33 @@ export class SharedTree
 	}
 }
 
+/**
+ * Get a {@link TreeBranch} from a {@link ITree}.
+ * @remarks The branch can be used for "version control"-style coordination of edits on the tree.
+ * @privateRemarks This function will be removed if/when the branching API becomes public,
```

Comment:
Honestly, these private remarks probably want to be public (standard remarks). `@privateRemarks` don't end up in our API docs, and I think this information is potentially useful to end-users experimenting with this API.

Reply:
I don't want to promise anything more than we need to regarding the future public API.

Reply:
The point isn't to make promises about what the API _will_ look like. It's to note to users that the current API shape is not what we expect it to look like long term.

Reply:
Sorry - I thought I had added a line that says "This API is subject to change...", and I did, but only on the second overload. Either way though, are we expected to write a disclaimer along with every alpha API we expose? I don't think it's a bad idea, but we should probably have something standard if so.

Reply:
I think it depends on the kind of experimental API. In some cases, we'll be adding an API that we actually potentially expect to promote more or less as-is. In other cases, we're forced to do things like this (adding a free function, rather than a method on some existing type) to avoid breaking changes. In the latter case, I think it's helpful to note that API is exposed in a less-than-ideal format to avoid breaking changes, so users are prepared for the shape to change when it gets promoted.

I don't think it's critical, but I think it helps set some useful expectations for users and for other devs on the team.